accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/impl/AcToolInstallHookServiceImpl.java

```diff
@@ -27,17 +29,24 @@
 @Component
 public class AcToolInstallHookServiceImpl implements AcToolInstallHookService {
 
+    public static final String ACL_HOOK_PATHS="actool.hook.config.paths";
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private AcInstallationServiceInternal acInstallationService;
 
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private ConfigFilesRetriever configFilesRetriever;
 
     @Override
-    public PersistableInstallationLogger installYamlFilesFromPackage(Archive archive, Session session, ProgressTrackerListener progressTrackerListener)
+    public PersistableInstallationLogger installYamlFilesFromPackage(VaultPackage  vaultPackage, Session session, ProgressTrackerListener progressTrackerListener)
             throws Exception {
+        Archive archive=vaultPackage.getArchive();
+        String configFilePaths=vaultPackage.getProperties().getProperty(ACL_HOOK_PATHS);
```

Comment:
I wanted to allow more than one property, but the API doesnt seem to provide a way to obtain all properties of the package so I can filter them.
https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/org/apache/jackrabbit/vault/packaging/PackageProperties.html

Reply:
Just use a comma separated string and split by comma and then make clear in the documentation that comma is not allowed in the regex pattern itself. Also name it rather configFilePathPatterns

Reply:
doesnt this just limit the user when they could just use `(a)|(b)|(c) `?

Reply:
Putting everything in one big regex is feasible but usually not very readable. For the same reason other configuration parameter allow multiple regexes: e.g. https://jackrabbit.apache.org/filevault/filter.html#Include_and_Exclude_Elements.

Retrieving all properties of a package is possible via `getMetaInf().getProperties()` (which just returns a Properties object).

Reply:
thanks! I missed that from the API. I've updated the PR.
you can pass multiple properties named actool.hook.paths.patterns.XXX to provide multiple patterns. I removed _config_  from it since it made it a bit too long.